### PR TITLE
Bug fix and Improvements

### PR DESCRIPTION
**Pull request by Tech-max-debug**
# Contribution checklist

## Brief description
- Fixed sending empty message on entire app.
- Fixed empty card creation on boards module.
- Fixed Image and attachment issue by reusing the attachments collections for both product images and attachments.
- Fixed profile icon alignment on templates page tracker module.

## Checklist

* [ ] - UI test added to added/changed functionality?
* [x] - Screenshot is added to PR if applicable ?
* [ ] - Does a local formatting is applied (rush format)
* [ ] - Does a local svelte-check is applied (rush svelte-check)
* [ ] - Does a local UI tests are executed [UI Testing](../tests/readme.md)
* [x] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [x] - Does it well tested?
* [x] - Tested for Chrome.
* [x] - Tested for Safari.
* [x] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [x] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

## Contributor requirements

* [ ] - Sign-off is provided. [DCO](https://github.com/apps/dco)
* [ ] - GPG Signature is provided. [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

### CODE REVIEW

**Comment by Tech-max-debug** [comment]
Alignment issue before:
Screenshot 2024-03-31 at 11 10 03 AM

Fixed:
Screenshot 2024-03-31 at 11 11 09 AM

File: plugins/attachment-resources/src/components/Photos.svelte [hunk 1]
```diff
@@ -62,6 +62,15 @@
     }
   }
 
+  async function deleteImage (id: string) {
```
Comment (aonnikov) [inline plugins/attachment-resources/src/components/Photos.svelte]: I don't see any usages of this function, do we need it?

File: packages/text-editor/src/components/TextEditor.svelte [hunk 2]
```diff
@@ -62,8 +62,11 @@
     }
   }
   export function submit (): void {
-    content = editor.getHTML()
-    dispatch('content', content)
+    const content = editor.getHTML().replace(/<[^>]*>/g, '');
```
Comment (aonnikov) [inline packages/text-editor/src/components/TextEditor.svelte]: I think this check is inaccurate, there can be elements that don't have inner content, such as images. For such content the content will be empty.

File: plugins/attachment-resources/src/components/Photos.svelte (hunk 1 above)
Comment (Tech-max-debug) [reply plugins/attachment-resources/src/components/Photos.svelte]: I was thinking of implementing the media delete functionality but realised that it's not really needed so skipped it.

File: packages/text-editor/src/components/TextEditor.svelte (hunk 2 above)
Comment (Tech-max-debug) [reply packages/text-editor/src/components/TextEditor.svelte]: Yes, I've already added a check on the submit button which verifies for an empty message, so yeah, this check can be omitted.

**Comment by aonnikov** [comment]
Fixes make sense, but there is no DCO

**Comment by Tech-max-debug** [comment]
Sign-off DCO is provided now.

**Review by aonnikov — APPROVED** [review]

**Pull request merged**